base/client.js

```diff
@@ -250,4 +252,9 @@ const hasNecessaryFields = error =>
   (typeof error.name === 'string' || typeof error.errorClass === 'string') &&
   (typeof error.message === 'string' || typeof error.errorMessage === 'string')
 
+const generateConfigErrorMessage = errors =>
+`Bugsnag configuration error\n${map(errors, (err) => `"${err.key}" ${err.message} \n    got ${stringify(err.value)}`).join('\n\n')}`
```

Comment:
If you're golfing, there's a bit of whitespace in this message, if not, this is fine :)